src/web/app/pages-help/instructor-help-page/instructor-help-page.component.ts

```diff
@@ -30,6 +35,19 @@ export class InstructorHelpPageComponent implements OnInit {
     }
   }
 
+  /**
+   * Scrolls to the section passed in
+   */
+  scroll(section: string): void {
+    if (this.bodyRef !== undefined) {
+      const el: any = Array.prototype.slice
+          .call(this.bodyRef.nativeElement.childNodes).find((x: any) => x.id === section);
+      if (el !== undefined) {
+        el.scrollIntoView();
```

Comment:
This causes the heading to be hidden behind the navbar. May need some offset.

Reply:
Offset by 50px, looks good to me now :)